unity/Assets/Resources/version.txt

```diff
@@ -1 +1 @@
-1.5.0a
```

Comment:
Please change this back (or change to 1.6.0.a).

Reply:
Changed to 1.6.0a (without the last dot)